OrbitGl/LiveFunctionsDataView.cpp

```diff
@@ -302,17 +303,12 @@ void LiveFunctionsDataView::DoFilter() {
 
 //-----------------------------------------------------------------------------
 void LiveFunctionsDataView::OnDataChanged() {
-  size_t numFunctions = Capture::GVisibleFunctionsMap.size();
-  m_Indices.resize(numFunctions);
-  for (size_t i = 0; i < numFunctions; ++i) {
-    m_Indices[i] = i;
-  }
-
   m_Functions.clear();
-  for (auto& pair : Capture::GVisibleFunctionsMap) {
-    const ULONG64& address = pair.first;
-    FunctionInfo* func = Capture::GVisibleFunctionsMap[address];
-    m_Functions.push_back(func);
+  size_t functions_count = Capture::GSelectedInCaptureFunctions.size();
+  m_Indices.resize(functions_count);
+  for (size_t i = 0; i < functions_count; ++i) {
+    m_Indices[i] = i;
```

Comment:
If you go for renaming [m_Functions](https://github.com/google/orbit/pull/916/files#r466394460), maybe also go for m_Indices to indices_.

Reply:
Done, thanks!